lms/templates/wiki/includes/article_menu.html

```diff
@@ -1,15 +1,17 @@
 ## mako
+<%page expression_filter="h"/>
 <%!
   from django.core.urlresolvers import reverse
   from django.utils.translation import ugettext as _
+  from openedx.core.djangolib.markup import Text, HTML
   from wiki.core.permissions import can_change_permissions
 %>
 
 <li class="${"active" if selected_tab == "view" else ""}">
   <a href="${reverse('wiki:get', kwargs={'article_id' : article.id, 'path' : urlpath.path})}">
     <i class="icon fa fa-eye"></i>
     ${_("View")}
-    ${_("{span_start}(active){span_end}").format(span_start="<span class='sr'>", span_end="</span>") if selected_tab == "view" else ""}
+    ${Text(_("{span_start}(active){span_end}")).format(span_start=HTML("<span class='sr'>"), span_end=HTML("</span>")) if selected_tab == "view" else ""}
```

Comment:
@bderusha: This original PR looks ok to me, so I am going to dig deeper to see what is going on here.